drivers/can/can_common.c

```diff
@@ -160,6 +160,30 @@ static int update_sampling_pnt(uint32_t total_tq, uint32_t sample_pnt,
 		sample_pnt - sample_pnt_res;
 }
 
+/**
+ * @brief Get the sample point location for a given bitrate
+ *
+ * @param  bitrate The bitrate in bits/second.
+ * @return The sample point in permille.
```

Comment:
fwiw the inconsistent spelling (permill/permille) is triggering me ;)

Reply:
I had not spotted those, thanks. I've changed them all to "permille".